src/uint256.cpp

```diff
@@ -22,7 +22,7 @@ std::string base_blob<BITS>::GetHex() const
 {
     char psz[sizeof(data) * 2 + 1];
     for (unsigned int i = 0; i < sizeof(data); i++)
-        sprintf(psz + i * 2, "%02x", data[sizeof(data) - i - 1]);
+        snprintf(psz + i * 2, 3, "%02x", data[sizeof(data) - i - 1]);
```

Comment:
For others http://www.cplusplus.com/reference/cstdio/snprintf/

> A terminating null character is automatically appended after the content written.

Hence,  2 characters + 1 null character.